technical-reports/format/composite-types.md

```diff
@@ -169,46 +235,221 @@ For example, imagine Sketch wanted to export all layer styles from a Sketch docu
+
+<aside class="example" title="Gradient token example">
+
+```json
+{
+  "blue-to-red": {
+    "type": "gradient",
+    "value": [
+      {
+        "color": "#0000ff",
+        "position": 0
+      },
+      {
+        "color": "#ff0000",
+        "position": 1
+      }
+    ]
+  }
 }
 ```
 
+Describes a gradient that goes from blue to red:
+
+<div style="height: 2rem; background: linear-gradient(90deg, #0000ff, #ff0000);"></div>
+
 </aside>
 
-The downside is of course that teams need to set up and maintain these configurations themselves.
+<aside class="example" title="Gradient token with omitted start stop example">
 
-### GUIs
+```json
+{
+  "mostly-yellow": {
+    "type": "gradient",
+    "value": [
+      {
+        "color": "#ffff00",
+        "position": 0.666
+      },
+      {
+        "color": "#ff0000",
+        "position": 1
+      }
+    ]
+  }
+}
+```
+
+Describes a gradient that is solid yellow for the first 2/3 and then fades to red:
+
+<div style="height: 2rem; background: linear-gradient(90deg, #ffff00 66.6%, #ff0000);"></div>
 
-Since composite types are ultimately composed of the core design token types, a design tool could automatically display an appropriate <abbr title="User Interface">UI</abbr> for creating, editing, or previewing them.
+</aside>
+
+<aside class="example" title="Gradient token using references example">
+
+```json
+{
+  "brand-primary": {
+    "type": "color",
+    "value": "#99ff66"
+  },
+
+  "position-end": {
+    "value": 1
+  },
 
-For instance, the color pair example above consists of 2 colors, one with the key `foreground` and the other with the key `background`. A design tool could therefore display two color picker widgets using those keys as the respective labels.
+  "brand-in-the-middle": {
+    "type": "gradient",
+    "value": [
+      {
+        "color": "#000000",
+        "position": 0
+      },
+      {
+        "color": "{brand-primary}",
+        "position": 0.5
+      },
+      {
+        "color": "#000000",
+        "position": "{position-end}"
+      }
+    ]
+  }
+}
+```
+
+Describes a color token called "brand-primary", which is referenced as the mid-point of a gradient is black at either end.
+
+<div style="height: 2rem; background: linear-gradient(90deg, #000000, #99ff66, #000000);"></div>
+
+</aside>
+
+<div class="issue" data-number="101" title="Gradient type feedback">
+  Is the current specification for gradients fit for purpose? Does it need to also specify the type of gradient (.e.g linear, radial, concial, etc.)?
+</div>
+
+## Typography
+
+Represents a typographic style. The type property must be set to the string “typography”. The value must be an object with the following properties:
+
+- `fontName`: The typography's font. The value of this property must be a valid [font name](#font-name) or a reference to a font name token.
```

Comment:
In the interest of sticking to `kebab-case`, what about naming this `font-family`? Otherwise it’d be the only property in `typography` that doesn’t map 1:1 to CSS.

Also, being pedantic, it technically is referring to a family because the name would include weight + italic/regular

Reply:
Good point, thanks! @mkeftz made [the same observation](https://github.com/design-tokens/community-group/issues/102#issuecomment-1012610832) in #102.

I'll change it to `fontFamily` (at the last editor's call we decided to standardise on camel case, so I might as well change. it accordingly now).